src/components/Children/AddChild.js

```diff
@@ -19,57 +19,59 @@ export default class AddChild extends Component {
     return(
       <div className='add-child'>
         <form className='add-info'>
+          <label>Name:</label>
           <input className='add-input name'
                 type='text'
                 value={this.state.name}
-                placeholder='Name'
                 onChange={(e) => this.setState({
                   name: e.target.value
                 })}
           />
+          <label>Birthday:</label>
           <input className='add-input'
-                  type='text'
+                  type='date'
                   value={this.state.childBirthday}
-                  placeholder='Birthday'
                   onChange={(e) => this.setState({
                     Birthday: e.target.value
                   })}
           />
+          <label>Arrival Date:</label>
           <input className='add-input'
-                  type='text'
+                  type='date'
                   value={this.state.childArrivalDate}
-                  placeholder='Arrival Date'
                   onChange={(e) => this.setState({
                     Arrival: e.target.value
                   })}
           />
+          <label>Insurance Info:</label>
           <input className='add-input'
                   type='text'
                   value={this.state.childInsurance}
-                  placeholder='Insurance'
                   onChange={(e) => this.setState({
                     Insurance: e.target.value
                   })}
           />
+          <label>Worker Name:</label>
           <input className='add-input'
                   type='text'
-                  value={this.state.childInsurance}
-                  placeholder='Worker Name'
+                  value={this.state.Worker_Name}
                   onChange={(e) => this.setState({
                     Worker_Name: e.target.value
                   })}
           />
+          <label>Worker Number:</label>
           <input className='add-input'
-                  type='text'
-                  value={this.state.childInsurance}
-                  placeholder='Worker Number'
+                  type='tel'
+                  value={this.state.Worker_Number}
+                  placeholder='xxx-xxx-xxxx'
```

Comment:
For a future feature, might be nice to see the phone number portion of the form get some logic so the user doesn't have to put in the "-"s? Unless that's built into the 'tel' type?